nltools/utils.py

```diff
@@ -218,10 +229,14 @@ def check_square_numpy_matrix(data):
     else:
         data = np.array(data)
 
-    if data.shape[0] != data.shape[1]:
-        raise ValueError('Make sure data is square.')
+    if len(data.shape) != 2:
```

Comment:
You now have a silent error if data.shape isn't len 2.  This could be problematic when the whole point of the function is to check if it's square.  Can you fix this, or revert back?  I don't understand why you made this change.